frontend/src/scenes/trends/personsModalLogic.ts

```diff
@@ -84,8 +84,8 @@ export function parsePeopleParams(peopleParams: PeopleParamType, filters: Partia
             { key: params.breakdown, value: breakdown_value, type: 'event' } as PropertyFilter,
         ]
     }
-    if (action?.properties) {
-        params.properties = { ...(flattenedPropertyGroup || {}), ...action.properties }
+    if (action?.properties && (action.properties?.length ?? 0) > 0) {
```

Comment:
Hmm, not sure if this doesn't introduce other new bugs. Getting very confusing..

See in the line above, where if we're dealing with a breakdown, we set `params.properties` to include the breakdown filter value, which is necessary for showing events only for the selected breakdown value.

And here, we're overriding that setting with properties on the action? (not sure if these are global filters or entity filters).

In any case, this feels a little bit better, as atleast `params.properties` are consistent now.

However, another case worth keeping in mind is that if `params.properties` originally were an OR property group, the flattening turns it into an AND, which will again return wrong results.

Reply:
Hm, I actually don't think this comment applies any more 

```
    // If breakdown type is cohort, we use breakdown_value
    // If breakdown type is event, we just set another filter
```

We might be able to get away with getting rid of the whole flattening nonsense